Update active-directory-conditional-access-technical-reference.md

efrene opened the pull request:
Suggesting the edits per CSS request.  See this blog for more information:  https://blogs.technet.microsoft.com/projectsupport/2017/12/21/project-online-now-with-conditional-access/

azurecla commented:
Hi __@bewillia__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@bewillia We're closing this PR because it appears to have only been a test.  Please comment if we should reopen.

efrene committed:
Update active-directory-conditional-access-technical-reference.md

Suggesting the edits per CSS request.  See this blog for more information:  https://blogs.technet.microsoft.com/projectsupport/2017/12/21/project-online-now-with-conditional-access/
